### DISCUSSION TITLE
Why I cant get my benefits but I‘m approved two days ago？

### DISCUSSION THREAD

**Discussion by TmkShink** [community/community#133248] [GitHub Education]
### Hi everyone,

Why I cant get my benefits but I‘m approved two days ago？

**Comment by davevad93** [comment]
Hi @Skylarpopgirl ,

Take a look at this [comment](https://github.com/orgs/community/discussions/133168#discussioncomment-10125818).

**Reply by tszdabee** [reply]
@Skylarpopgirl It usually takes around 1-2 business days for the academic benefits of the GitHub Student Developer Pack to show up after being approved. If you still haven't received your benefits after approval within 5 days, please raise another discussion so it can be addressed.

[How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
